pom.xml

```diff
@@ -42,7 +42,49 @@
                 <directory>target/generated-resources</directory>
             </resource>
         </resources>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
+                    <version>3.3.0</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-clean-plugin</artifactId>
+                    <version>2.5</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-install-plugin</artifactId>
+                    <version>2.5.1</version>
```

Comment:
Shouldn't the version fixes here come from the parent pom? If you have a look at next branch, is that still an issue?

Reply:
I added `requirePluginVersions` first and then added whatever plugin it was complaining about. Let me check...

Reply:
AFAICT, on the next branch, all plugin versions were already managed :+1: